lib/redis_memo/memoize_query.rb

```diff
@@ -13,6 +13,8 @@ module RedisMemo::MemoizeQuery
     # after each record save
     def memoize_table_column(*raw_columns, editable: true)
       RedisMemo::MemoizeQuery.using_active_record!(self)
+      return if ENV["REDIS_MEMO_DISABLE_#{self.table_name.upcase}"] == 'true'
```

Comment:
Can we just use the env var `REDIS_MEMO_DISABLE_QUERY_..` here as well? In what case would we want to disable memoizing the columns of a table using `REDIS_MEMO_DISABLE_*`, but not also want to disable all queries on the table (or vice versa)?

Reply:
`REDIS_MEMO_DISABLE_*` will also disable all the cached method referencing those memoized table columns; so it'd disable more things.